### PR TITLE
Add version to repo and narrative_status/

### PR DESCRIPTION
**Pull request by kkellerlbl**

### CODE REVIEW

**Review by bio-boris — APPROVED** [review]
Could also do a git commit hash like https://ci.kbase.us/services/auth/

**Review by bio-boris — APPROVED** [review]
Could also do a version.py https://packaging.python.org/guides/single-sourcing-package-version/

File: app.py [hunk 1]
```diff
@@ -17,6 +17,8 @@
 from typing import Dict, List, Optional
 from types import FrameType
 
+VERSION = "0.0.1"
```
Comment (sychan) [inline app.py]: I've already tagged it as 0.9.0 - we're putting this into prod, so its too late to call it 0.0.1!

**Comment by kkellerlbl** [comment]
> Could also do a git commit hash like https://ci.kbase.us/services/auth/

Yeah, I didn't feel like doing that much work yet.  :)  As a first pass this is fine.

File: app.py (hunk 1 above)
Comment (kkellerlbl) [reply app.py]: Fixed version to 0.9.0.

**Pull request merged**